application/templates/cronjob.yaml

```diff
@@ -31,10 +31,17 @@ spec:
 {{ end }}
   jobTemplate:
     spec:
+{{- if $job.activeDeadlineSeconds }}
+      activeDeadlineSeconds: {{ $job.activeDeadlineSeconds }}
+{{ end }}
       template:
         metadata:
           labels:
           {{- include "application.labels" $ | nindent 12 }}
+          {{- with $job.additionalPodAnnotations }}
```

Comment:
```suggestion
          {{- with $job.podAnnotations }}
```
is enough

Reply:
I also added a property `additionalPodLabels`, now isn't it better to keep both with the same name format ?

Reply:
I don't really care, as you'd like

Reply:
I prefer to have them with the same format :)